packages/react-positioning/src/usePopper.ts

```diff
@@ -1,65 +1,34 @@
-import { useEventCallback, useIsomorphicLayoutEffect, useFirstMount, canUseDOM } from '@fluentui/react-utilities';
+import { computePosition, hide as hideMiddleware, arrow as arrowMiddleware } from '@floating-ui/dom';
+import type { Middleware, Strategy } from '@floating-ui/dom';
 import { useFluent } from '@fluentui/react-shared-contexts';
-import * as PopperJs from '@popperjs/core';
+import { canUseDOM, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
+import { useEventCallback } from '@fluentui/react-utilities';
 import * as React from 'react';
-
-import { isIntersectingModifier } from './isIntersectingModifier';
+import type { FloatingUIOptions, PositioningProps, PositioningVirtualElement } from './types';
+import { getPlacement } from './utils/positioningHelper';
+import { useCallbackRef } from './utils/useCallbackRef';
 import {
-  getScrollParent,
-  applyRtlToOffset,
-  getPlacement,
-  getReactFiberFromNode,
-  getBoundary,
-  useCallbackRef,
-  getBasePlacement,
-} from './utils/index';
-import type { PopperVirtualElement, PopperOptions, PositioningProps } from './types';
-
-type PopperInstance = PopperJs.Instance & { isFirstRun?: boolean };
-
-interface UsePopperOptions extends PositioningProps {
+  offset as offsetMiddleware,
+  shift as shiftMiddleware,
+  flip as flipMiddleware,
+  coverTarget as coverTargetMiddleware,
+  maxSize as maxSizeMiddleware,
+  intersecting as intersectingMiddleware,
+} from './middleware/index';
+import { getScrollParent } from './utils/getScrollParent';
+import debounce from './utils/debounce';
```

Comment:
```suggestion
import { debounce } from './utils/debounce';
```

nit: We use named exports